### ISSUE TITLE
[BUG] Microsoft Bookings Connector Trigger - Dropdown Custom Question: No 'Answer' Field

### ISSUE THREAD

**Issue by ryanedshannon** [microsoft/PowerPlatformConnectors#2918]
### Type of Connector

Certified Connector

### Name of Connector

Microsoft Bookings

### Describe the bug

When an appointment is made in bookings, the trigger no longer provides the 'Answer' value for multiple choice questions. It passes a 'selected option' integer and an array of the choice options, however up until now we have used the more convenient 'Answer' string without issue.

Was this an intentional change or a bug? I need to know whether I need to rework all of our bookings flows to accommodate this.

### Is this a security bug?

No, this is not a security bug

### What is the severity of this bug?

Severity 3 - Minor issue

### To Reproduce

Create an appointment with a custom multiple choice field, the trigger will no longer have an 'Answer' for that field.

### Expected behavior

An 'Answer' string that gives the selected value.

### Environment summary

Power Automate Cloud Flow

### Additional context

I can't find documented changes for this connector.

**Comment by ryanedshannon** [comment]
I am also seeing that the 'update appointment' trigger now fires at the same time of appointment creation.

**Comment by javierj-icaba** [comment]
> I am also seeing that the 'update appointment' trigger now fires at the same time of appointment creation.

Yes! And also the UPDATE connector creates a new appointment, but it does NOT trigger the DELETE event for the previous one, so if you are exporting to Excel, the appointment is duplicated: the old one plus the new one.
